src/backend/distributed/commands/utility_hook.c

```diff
@@ -729,10 +729,18 @@ UndistributeDisconnectedCitusLocalTables(void)
 		 * undistribute it via cascade. Here, instead of first dropping foreing
 		 * keys then undistributing the table, we just set cascadeViaForeignKeys
 		 * to true for simplicity.
+		 *
+		 * We suppress notices messages not to be too verbose. On the other hand,
+		 * as UndistributeTable moves data to a new table, we want to inform user
+		 * as it might take some time.
 		 */
+		ereport(NOTICE, (errmsg("undistributing table %s since it is no longer "
```

Comment:
This sounds a little too technical. Also, these tables are already not distributed -- just added to metadata as #4573. 

So maybe:

```
Removing table %s from metadata and converting it to a local table as it is not connected to any distributed or reference tables via foreign key.
```

or even just:
```
Removing table %s from metadata as it is not connected to any distributed or reference tables via foreign key.
```
Do you have a preference?

Reply:
second one sounds better to me

Reply:
sounds good

Reply:
adressed now